### ISSUE TITLE
getUser - misspell

### ISSUE THREAD

**Issue by Mshq0** [murka/outlinevpn-api#7]
Using the library, I noticed a problem when executing this function. When using getUser() does not work with a third-party certificate, although the rest of the functions are fine! After looking at the code, I noticed an oversight, instead of the usual fetch, there should be a call from the class. So you should just add "this." with fetch

**Before**
```
fetch(...)
```
**After**
```
this.fetch(...)
```

Hello from Russia, I used translator.

**Comment by murka** [comment]
hey, what error you got? I have no some problem with this function. Do you provide certSha256 in options?

**Comment by Mshq0** [comment]
> hey, what the error you got? I have no some problem with this function. Do you provide certSha256 in options?

Yes, he pointed out that he was complaining about it. It started working as soon as I formatted the function code


here's how I fixed the bug:
```js
    getUser(id) {
        return __awaiter(this, void 0, void 0, function* () {
            const response = yield this.fetch({ url: `${this.apiUrl}/access-keys/${id}`, method: 'GET' });
            if (response.ok) {
                return JSON.parse(response.body)
            }
            return null;
        });
    }
```

**Comment by Mshq0** [comment]
Now, I used test code that looks like this:
`const { OutlineVPN } = require('outlinevpn-api')
const outline = new OutlineVPN({
    apiUrl: '',
    fingerprint: ''
})

async function main() {
    const users = await outline.getUsers()
    console.log(users)

    const user = await outline.getUser('0')
    console.log(user)
}

main()`

getUsers works and gives me data about users, but getUser does not work and gives me this error:
`node:internal/deps/undici/undici:12500
      Error.captureStackTrace(err, this);
            ^

TypeError: fetch failed
    at node:internal/deps/undici/undici:12500:13
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  [cause]: Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1674:34)
      at TLSSocket.emit (node:events:519:28)
      at TLSSocket._finishInit (node:_tls_wrap:1085:8)
      at ssl.onhandshakedone (node:_tls_wrap:871:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
}`



> > hey, what the error you got? I have no some problem with this function. Do you provide certSha256 in options?
> 
> Yes, he pointed out that he was complaining about it. It started working as soon as I formatted the function code
> 
> here's how I fixed the bug:
> 
> ```js
>     getUser(id) {
>         return __awaiter(this, void 0, void 0, function* () {
>             const response = yield this.fetch({ url: `${this.apiUrl}/access-keys/${id}`, method: 'GET' });
>             if (response.ok) {
>                 return JSON.parse(response.body)
>             }
>             return null;
>         });
>     }
> ```

**Comment by SerjNerj** [comment]
@murka i'd like to fix it but i have not permissions to create a branch or commit in your repo

**Comment by murka** [comment]
@SerjNerj you can fork the repo and keep changes.

**Comment by murka** [comment]
@Mshq0, fixed on outlinevpn-api@2.4.0
